app/src/main/res/drawable/ic_radio_button_unchecked.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we have one drawable with a a `selector` representing the entirety of the RadioButton?

Reply:
Can you animate the state change then?

Reply:
Today is the day we both learn something cool: AnimatedVectorDrawable has the ability to animate between states.

If we feel like not learning something today, we can go with the solution we have today (which is totally fine), but using AnimatedVectorDrawable to animate between states could be freakishly cool actually!